cms/templates/activation_complete.html

```diff
@@ -9,8 +9,8 @@
 <div class="wrapper-mast wrapper sr">
   <header class="mast">
     <h1 class="page-header">
-      ${_("{studio_name} Account Activation").format(
-        studio_name=Text(settings.STUDIO_SHORT_NAME)
+      ${Text(_("{studio_name} Account Activation")).format(
+        studio_name=Text(settings.STUDIO_SHORT_NAME),
```

Comment:
@dan-f Your explanation was much more brief than mine:
http://draft-safe-templates-2016-03-23.readthedocs.org/en/latest/conventions/safe_templates.html#why-do-i-need-text-with-html
I'll need to determine if I should cut mine down, or if it is useful for someone who really wants to know what's going on. :)
FYI: @douglashall